src/SDKs/CognitiveServices/dataPlane/Search/BingAutoSuggest/BingAutoSuggest/Microsoft.Azure.CognitiveServices.Search.AutoSuggest.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+  
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Search.AutoSuggest</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services AutoSuggest API.</Description>
+    <VersionPrefix>1.2.0</VersionPrefix>
```

Comment:
Is this is a preview package or a stable package.
Versioning must start from `0.9.0-preview` for preview versions and `1.0.0` for stable versions